test/test_performance.py.in

```diff
@@ -75,6 +79,14 @@ def _raw_to_png(csv_data, png_path):
 
     plt.savefig(png_path)
 
+    dataframe['data_received Mbits/s'] = dataframe['data_received'] / 1024
```

Comment:
There's a magic number 1024 here but I see 1e3 above. And several more 1024 instances. I'm not familiar with the underlying units but being a little clearer/consistent would be useful. Maybe a helper function or constant  for clarity would be helpful.